### PR TITLE
The email address of the sender is hidden issue #6980

### PR DESCRIPTION
**Pull request by g-fantini**
The email address of the sender is hidden #6980

The email address of the sender is hidden because handled as HTML tag

#### Issue
In the Email Detail view, if the email is not imported the sender email address is hidden, because considered a HTML tag [email@test.com](mailto:email@test.com)

#### Expected Behavior
The email address should be visible
image

#### Actual Behavior
the sender email address is hidden
image

## Description

The email address of the sender is hidden #6980

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: modules/Emails/include/DetailView/EmailsDetailView.php [hunk 1]
```diff
@@ -96,6 +96,8 @@ public function populateFields()
         if (empty($this->focus->from_addr_name)) {
             $this->focus->from_addr_name = $this->focus->from_addr;
         }
+        //Prevent Email address to be handled as HTML tag
```
Comment (samus-aran) [inline modules/Emails/include/DetailView/EmailsDetailView.php]: Can we not htmlentities it instead of a clunky str_replace?

**Comment by willrennie** [comment]
@g-fantini would be great if you can review the comments and adjust your PR, thanks!

**Comment by samus-aran** [comment]
@g-fantini Can you provide an better solution?

**Comment by Mausino** [comment]
@samus-aran will great if this issue will in next patch :)  for me this last patch working

Also question: Why on list view can't show as in detail view the Name + email ?

Because if email address exists, show there only Name, and if not only email address.. and if you filter by TO or FROM you see only email address also in situation if exist email in database. 
This will help us users very much :)

**Comment by cameronblaikie** [comment]
Assessed :+1:

**Comment by cameronblaikie** [comment]
Assessed :+1:

**Pull request merged**